tsl/test/expected/compression_ddl.out

```diff
@@ -1039,3 +1039,94 @@ SELECT * FROM test_defaults ORDER BY 1,2;
  Mon Jan 01 00:00:00 2001 PST |         1 |    | 42
 (3 rows)
 
+-- test dropping columns from compressed
+CREATE TABLE test_drop(f1 text, f2 text, f3 text, time timestamptz, device int);
+SELECT create_hypertable('test_drop','time');
+psql:include/compression_alter.sql:176: NOTICE:  adding not-null constraint to column "time"
+    create_hypertable    
+-------------------------
+ (14,public,test_drop,t)
+(1 row)
+
+ALTER TABLE test_drop SET (timescaledb.compress,timescaledb.compress_segmentby='device');
```

Comment:
Think should add test with non-time column  timescaledb.compress_orderby = 'dummy col' and then test dropping it
ALTER TABLE test_drop DROP COLUMN <dummy_col>.

Reply:
done